source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+
+* **Recovery Point Objective (RPO)**
+
+  There is a big distinction between the Recovery Point and the Recovery Time for a database infrastructure. The database can be available, but not to the exact state that it was when it became unavailable. That is where Recovery Point comes in. The question to ask here is "How much data can the business lose during a database outage?" All businesses have their own requirements here yet it is always the goal to never sustain any data loss. But this is framed in the worst case scenario, how much data could be lost and the business maintains the ability to continue.
+
+* **Disaster recovery**
+
+  RTO and RPO are great for unplanned outages or small scale hiccups to the infrastructure. Disaster recovery is a major large scale outage not strictly for the database infrastructure. How capable is the business of restarting operations with the assumption that all resources are completely unavailable in the main availability zone? The assumption here is that there is no viable restoration point or time that aligns with the business requirements. While each disaster recovery scenario is unique based on available infrastructure, backup strategy and technology stack, there are some common threads for any scenario.
+
+The described solution **helps improve uptime**. It will help you to significantly reduce both RPO and RTO. Given the tightly coupled cluster solution approach, the failure of a single node will not result in service interruption. 
+
+Increasing the number of nodes will also improve the cluster resilience by the formula:
+
+.. code-block:: text
+
+   F = (N -1) / 2
+
+where:
+
+- ``F`` is the number of admissible failures
+- ``N`` is the number of nodes in the cluster.
+
+.. admonition:: Examples:
+ 
+  - In a cluster of 5 nodes, F = (5 - 1)/2 = 2. 
+    The cluster can support up to 2 failures. 
+
+  - In a cluster of 4 nodes, F = (4 - 1)/2 = 1. 
+    The cluster can support up to 1 failure. 
+
+This solution also allows for a more restrictive backup policy, dedicating a node to the backup cycle, which will help in keeping RPO low.
+
+As previously mentioned, disaster recovery is not covered by default by this solution. It will require an additional replication setup and controller. 
+
+Measurement and Monitoring
+--------------------------
+
+To ensure that database infrastructure is performing as intended or at its best, it is necessary to measure specific metrics and alert when some of these metrics are not in line with expectations. Periodic review of these measurements is also encouraged to promote stability and understand potential risks associated with the database infrastructure. 
```

Comment:
To ensure that database infrastructure is performing as intended or at its best, it

This seems to be a dangling modifier  - the fragment doesn't modify the "it". 

A perodic

Reply:
amended